test-basic/plan-builder-generated.js

```diff
@@ -2353,8 +2395,12 @@ describe('plan builder', function() {
     }); 
     it('xs.negativeInteger#1', function(done) {
         testPlan([p.xs.double(-1)], p.xs.negativeInteger(p.col("1")))
-          .then(function(response) { 
-            should(String(getResult(response).value).replace(/^ /, '')).equal('-1');
+          .then(function(response) {
+              if(serverConfiguration.serverVersion < 12) {
+                  should(String(getResult(response).value).replace(/^ /, '')).equal('-1');
+              } else {
+                  should(String(getResult(response).value).replace(/^ /, '')).equal(null);
```

Comment:
Why is there here for MarkLogic 12? Is it expected that this now returns null instead of -1?

Reply:
Since it was auto-generated I would say yes, but for server versions less than 12, it gives -1 so added the if else block.